src/model/versioning.test.ts

```diff
@@ -125,6 +125,7 @@ describe('Versioning', () => {
       'is happy with valid %s',
       (description) => {
         expect(Versioning.descriptionRegex1.test(description)).toBeTruthy();
+        expect(Versioning.descriptionRegex1.test(description?.slice(1))).toBeTruthy();
```

Comment:
Suggest adding a separate tests for `it('accepts valid semantic versions without v-prefix', () => {})`

Reply:
Done